package.json

```diff
@@ -73,6 +74,7 @@
     "rimraf": "^3.0.2",
     "semver": "^7.3.7",
     "split2": "^4.1.0",
+    "standard": "^17.1.0",
```

Comment:
Actually, looks like adding `standard` is making the linter fail. We already have `ts-standard` installed, so it would be better to import and use that instead.

Reply:
Okay, I THINK I got it!